config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-main-presubmits.yaml

```diff
@@ -208,6 +208,7 @@ presubmits:
     - ^main$
     labels:
       preset-dind-enabled: "true"
+      preset-cluster-api-provider-vsphere-e2e-config: "true"
```

Comment:
This is required otherwise some assumptions hard-coded in e2e.sh fail. e.g. jobs fails with `./hack/e2e.sh: line 71: GOVC_URL: unbound variable`

Reply:
I'm ok with fixing it like this for now.

Let's see how it looks like at the end and get rid of it here again so this jobs can get moved to the community prow :-)

Reply:
one option we can explore is to create a "preset-cluster-api-provider-vsphere-e2e-config-vcsim" that gives some fake values for the required variables.
The alternative is to use defaults in the e2e.sh script.